packages/e2e-tests/plugins/inner-blocks-prioritized-inserter-blocks/index.js

```diff
@@ -0,0 +1,81 @@
+( function () {
+	const { registerBlockType } = wp.blocks;
+	const { createElement: el } = wp.element;
+	const { InnerBlocks } = wp.blockEditor;
+
+    const divProps = {
+		className: 'product',
+		style: { outline: '1px solid gray', padding: 5 },
+	};
+
+    // Make it easier to select the block.
```

Comment:
What does this comment mean?

Reply:
It was indicating that without a placeholder within the innerblocks it can be difficult to select the block using e2e tests, especially using Puppeteer. Open to suggestions for improving the comment 🙇

Reply:
> without a placeholder within the innerblocks it can be difficult to select the block using e2e tests, especially using Puppeteer, so we use an image block which has a placeholder.